### PR TITLE
Add missing system includes for `LOCK_EX` and `struct timex`

### PR DESCRIPTION
**Pull request by kraj**

### CODE REVIEW

File: src/test/test-sizeof.c [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 #include <sys/resource.h>
 #include <sys/socket.h>
+#include <sys/timex.h>
 #include <sys/types.h>
 
 #define __STDC_WANT_IEC_60559_TYPES_EXT__
 #include <float.h>
 
 #include "time-util.h"
+#include "cpu-set-util.h"
```
Comment (yuwata) [inline src/test/test-sizeof.c]: Why did you change `sched.h` -> `cpu-set-util.h`??

File: src/test/test-fs-util.c [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
 #include <unistd.h>
+#include <sys/file.h>
```
Comment (bluca) [inline src/test/test-fs-util.c:3]: sort alphabetically (move one up)

File: src/test/test-sizeof.c (hunk 1 above)
Comment (kraj) [reply src/test/test-sizeof.c]: oversight. Removed in v2

**Review by bluca — APPROVED** [review]

File: src/test/test-sizeof.c [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include <sched.h>
```
Comment (bluca) [inline src/test/test-sizeof.c]: removing this breaks the build, any reason why it was taken out?

```
../src/test/test-sizeof.c: In function ‘main’:
../src/test/test-sizeof.c:108:14: error: ‘__cpu_mask’ undeclared (first use in this function); did you mean ‘__fd_mask’?
  108 |         info(__cpu_mask);
```

**Review by yuwata — APPROVED** [review]

File: src/test/test-sizeof.c (hunk 3 above)
Comment (kraj) [reply src/test/test-sizeof.c]: Another oversight :(. it seems to be a bad day. Fixed now.
